### ISSUE TITLE
Tutorial task-07 Error | ASSERTION FAILED

### ISSUE THREAD

**Issue by npcode15** [RIOT-OS/RIOT#10016]
Hello, I am trying to solve the task-07 (7.2 to be precise) from the tutorials.

I wrote a new function: rcv_pkt() and called it at the end of the main function.
Please refer to the following screenshot (of function code) for a better understanding.
![image](https://user-images.githubusercontent.com/25517038/45922083-3a6f4300-be88-11e8-9c81-7bdb58016121.png)

**Error:** Compilation succeeds with 'make all' but when I use the command 'make all term' I get the following error.
![image](https://user-images.githubusercontent.com/25517038/45922055-c59c0900-be87-11e8-9550-1457c7e2abb9.png)

I tried using make clean, touch to solve the problem of skewed clock but didn't succeed so far.

I appreciate any help and thanks in advance.

**UPDATE**: 
![image](https://user-images.githubusercontent.com/25517038/45979250-222c2f00-c014-11e8-8b69-b1024e439cf1.png)
**I was able to fix the issue of skewed clock. But, the execution still fails.**

@aabadie @dylad

**Comment by aabadie** [comment]
@npcode15, your question seems related to the Tutorial, maybe you should post [there](https://github.com/RIOT-OS/Tutorials/issues/new) ?

To give you an advice on your problem, you get a `FAILED ASSERTION` indeed, but if you look just above, the file and line where this happened is also given: `sys/net/gnrc/netreg/gnrc_netreg.c:49`. Could it be that you didn't initialize a message queue ?

**Comment by npcode15** [comment]
@aabadie, Sorry I should have posted there. But, I will open an issue and close it referencing this one. So, that the issue will searchable.

Regarding the solution, yes it worked. I missed the initialization somehow. But, I am getting another error (please look at the following image).
![image](https://user-images.githubusercontent.com/25517038/46004888-9e0c9280-c079-11e8-9611-bcbbc7fb8508.png)

I tried to initiate the UDP server from the main.c file. This was fine before. I have no idea what broke.
I even cloned a fresh copy but didn't help. Does anything spring to your mind?

**Current gnrc_minimal_code**
![image](https://user-images.githubusercontent.com/25517038/46005131-31de5e80-c07a-11e8-9fd5-da3847ded766.png)

**Comment by npcode15** [comment]
Nope, that was not the issue but I am getting a segmentation dump when I commented out line 126.
But, it turns out I do not need udp.c file because I subscribe to UDP packets. So, I was able to fix the issue for my purpose. But, it still fails.

![image](https://user-images.githubusercontent.com/25517038/46007743-71a84480-c080-11e8-97e0-b9677ed2f598.png)
**One possibility:** Because I am already subscribed to receive UDP and I am again trying to do so by starting UDP server. There is a conflict, hence the error. Just a guess.

Also, Should I close the issue?
